torch/_torch_docs.py

```diff
@@ -10565,9 +10565,9 @@ def merge_dicts(*dicts):
     device(type='cpu')
 """)
 
-add_docstr(torch.assert_async,
+add_docstr(torch._assert_async,
```

Comment:
Since this isn't added to torch.rst it won't appear.

I don't think assert_async() was added to torch.rst, either?

Reply:
No it wasn't. I guess it's fine for it to not show up now that it is underscored.